### DISCUSSION TITLE
What is the state with Vulkan backend?

### DISCUSSION THREAD

**Discussion by trsh** [LuisaGroup/LuisaCompute#155] [Q&A]
WIP / Maybe / ?

**Comment by Mike-Leo-Smith** [comment]
Hi, @trsh 

The Vulkan backend was suspended but should be resumed soon. @MaxwellGengYF made an initial implementation but encountered some bugs and performance issues. We now have a new master's student in the lab who wants to take it on. I'm also working on the new IR module, which should assist with SPIR-V generation.
